modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -98,56 +104,37 @@ PARTITION  LEADER  EPOCH  REPLICAS  LOG-START-OFFSET  HIGH-WATERMARK
 2          3       1      [0 1 3]   0                 672
 ----
 
-. Put all partition reassignments in a JSON file. For example, to change the replica set of partition 1 from `[0 1 2]` to `[3 1 2]` and change the replica set of partition 2 from `[0 1 3]` to `[2 1 3]`:
-+
-[,json]
-----
-{
-  "version": 1,
-  "partitions": [
-    {
-      "topic": "test",
-      "partition": 1,
-      "replicas": [
-        3,
-        1,
-        2
-      ]
-    },
-    {
-      "topic": "test",
-      "partition": 2,
-      "replicas": [
-        2,
-        1,
-        3
-      ]
-    }
-  ]
-}
-----
-
-. Reassign partitions with the `kafka-reassign-partitions.sh` script. This example uses `example.json` as the name of the JSON file:
+. Change partition assignments. For example, to change the replica set of partition 1 from `[0 1 2]` to `[3 1 2]`, and to change the replica set of partition 2 from `[0 1 3]` to `[2 1 3]`, run:
 +
 [,bash]
 ----
-kafka-reassign-partitions.sh --bootstrap-server localhost:9092,localhost:9093,localhost:9094,localhost:9095 --reassignment-json-file example.json --execute
-Current partition replica assignment
+rpk cluster partitions move test -p 1:3,1,2 -p 2:2,1,3
+NAMESPACE  TOPIC  PARTITION  OLD-REPLICAS     NEW-REPLICAS      ERROR
+kafka      test   1          [0-1, 1-1, 2-0]  [1-1, 2-0, 3-0]
+kafka      test   2          [0-0, 1-0, 3-1]  [1-0, 2-0, 3-1]
 
-{"version":1,"partitions":[{"topic":"test","partition":1,"replicas":[1,2,0],"log_dirs":["any","any","any"]},{"topic":"test","partition":2,"replicas":[3,1,0],"log_dirs":["any","any","any"]}]}
+Successfully began 2 partition movement(s).
 
-Save this to use as the --reassignment-json-file option during rollback
-Successfully started partition reassignments for test-1,test-2
+Check the movement status with `rpk cluster partitions move-status`, and see new assignments with `rpk topic describe -p TOPIC`.
+----
++
+or
++
+[,bash]
+----
+rpk cluster partitions move -p test/1:3,1,2 -p test/2:2,1,3
 ----
 
-. Verify that the reassignment is complete with the flags `--verify --preserve-throttles`:
+. Verify that the reassignment is complete with `move-status`:
```

Comment:
actually, this whole section like a duplicate to the statement above and a better version of it.  perhaps we can just remove the statement above .

> Check the movement status with `rpk cluster partitions move-status`, and see new assignments with `rpk topic describe -p TOPIC`.

Reply:
> Check the movement status with rpk cluster partitions move-status, and see new assignments with rpk topic describe -p TOPIC.

This is an output of of the move command. If this is confusing, we can just remove it from the example. Wdyt? @mattschumpert